content/governing-content/content-lifecycle/planning-content/content-planning-workshop/start.md

```diff
@@ -2,5 +2,19 @@
 layout: text/textblock
 section: Start with a user focus
 ---
-Bring your team together to create an overarching user need. You can do this by creating a user story. Doing this at the start will give the team a focus for the session. It puts the user at the centre of what you are planning.
+Bring your team together to create an overarching user need. You can do this by creating a user story. Doing this at the start will give the team a focus for the workshop. It puts the user at the centre of what you are planning.
```

Comment:
IMHO this would benefit with linking to the other content around user stories and user needs.

Reply:
Good point, will make that change.